### ISSUE TITLE
We should make animate_trajectory adapt to what fields are included in the tuple passed into it.

### ISSUE THREAD

**Issue by murrellb** [MurrellGroup/ProtPlot.jl#46]
Eg. if it has amino acid logits, make the lower panel, but otherwise don't. If it only has Xt, then only make the left panel. Etc.
